index.ts

```diff
@@ -104,13 +186,12 @@ async function main() {
       ethers.ZeroAddress,
     ],
   );
-  const safeAddress =
-    await contracts.proxyFactory.createProxyWithNonce.staticCall(
-      contracts.singleton,
-      setup,
-      SAFE_SALT_NONCE,
-    );
```

Comment:
This was removed in favour of getCreate2Address (which should work in all cases - instead of only pre-deployment).